src/js/components/AppListComponent.jsx

```diff
@@ -186,7 +187,10 @@ var AppListComponent = React.createClass({
 
     if (filterText != null && filterText !== "") {
       nodesSequence = nodesSequence
-        .filter(app => app.id.indexOf(filterText) !== -1);
+        .filter(app => {
+          return score(app.id, filterText) > 0.025;
```

Comment:
Why is this needed? It leads to unexpected behavior.

Before (like it was), I see all result (ranked with the sorting later):
![score-before](https://cloud.githubusercontent.com/assets/859154/12449096/231fe1dc-bf7b-11e5-9024-315fd333a57b.png)
:+1: 

After - one result is hidden, I don't expect this:
![score-after](https://cloud.githubusercontent.com/assets/859154/12449106/38dbcdba-bf7b-11e5-9a44-3edfd13e5534.png)
:-1: 

Let's stick with indexOf() and keep it simple.